Api/Services/RestaurantService.cs

```diff
@@ -202,83 +202,88 @@ public async Task<List<RestaurantSummaryVM>> GetMyRestaurantsAsync(User user)
         }
 
         /// <summary>
-        /// Add the given employee to the given restaurant, acting as the given employer (restaurant owner)
+        /// Add employees to the given restaurant, acting as the given employer (restaurant owner)
         /// </summary>
         /// <param name="restaurantId">ID of the restaurant to add the employee to</param>
         /// <param name="employerId">ID of the current user (restaurant owner)</param>
         /// <returns>The bool returned inside the result does not mean anything</returns>
-        public async Task<Result<bool>> AddEmployeeAsync(AddEmployeeRequest request, int restaurantId, string employerId)
+        public async Task<Result<bool>> AddEmployeeAsync(List<AddEmployeeRequest> listRequest, int restaurantId, string employerId)
         {
-            if (!request.IsBackdoorEmployee && !request.IsHallEmployee)
+            foreach (var request in listRequest)
             {
-                return new List<ValidationResult>
+
+                var result = await validationService.ValidateAsync(request);
+                if (!result.IsValid)
                 {
-                    new($"Employee must have at least one role",
-                        [nameof(request.IsBackdoorEmployee), nameof(request.IsHallEmployee)])
-                };
-            }
+                    return result;
+                }
 
-            var restaurantOwnerId = await context.Restaurants
-                .Where(r => r.Id == restaurantId)
-                .Select(r => r.Group!.OwnerId)
-                .FirstOrDefaultAsync();
-            if (restaurantOwnerId is null)
-            {
-                return new List<ValidationResult>
+                var restaurantOwnerId = await context.Restaurants
+                    .Where(r => r.Id == restaurantId)
+                    .Select(r => r.Group!.OwnerId)
+                    .FirstOrDefaultAsync();
+
+
+                if (restaurantOwnerId is null)
                 {
-                    new($"Restaurant with ID {restaurantId} not found")
-                };
-            }
+                    return new ValidationFailure
+                    {
+                        PropertyName = nameof(restaurantOwnerId),
```

Comment:
Nie można podawać nazw zimennych lokalnych tu, bo to dla fronta. trzeba tu podać odpowiednie property requesta. Ale w tym przypadku to problem z parametrem w ścieżce, więc trzeba podać null